addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -537,6 +522,277 @@
+                </we-button>
+                <we-button title="Dotted" data-customize-css-property="dotted">
+                    <div class="o_we_fake_img_item o_we_border_preview" style="border-style: dotted;"/>
+                </we-button>
+                <we-button title="Double" data-customize-css-property="double">
+                    <div class="o_we_fake_img_item o_we_border_preview" style="border-style: double; border-left: none; border-right: none;"/>
+                </we-button>
+            </we-select>
+            <we-colorpicker data-customize-css-property=""
+                            data-css-property="border-color"
+                            data-selector-text="a[href].btn-secondary, a[href].btn-outline-secondary, a[href].btn-fill-secondary"
+                            data-no-transparency="true"
+                            data-color-prefix="border-"/>
+        </we-row>
+        <!-- SEPARATORS -->
+        <we-row string="Separators" class="o_design_tab_title">
+            <we-input data-customize-css-property=""
+                    data-css-property="border-top-width"
+                    data-selector-text=".s_hr > hr"
+                    data-unit="px"/>
+            <we-select data-selector-text=".s_hr > hr" data-css-property="border-top-style">
+                <we-button title="Solid" data-customize-css-property="solid">
+                    <div class="o_we_fake_img_item o_we_border_preview" style="border-style: solid;"/>
+                </we-button>
+                <we-button title="Dashed" data-customize-css-property="dashed">
+                    <div class="o_we_fake_img_item o_we_border_preview" style="border-style: dashed;"/>
+                </we-button>
+                <we-button title="Dotted" data-customize-css-property="dotted">
+                    <div class="o_we_fake_img_item o_we_border_preview" style="border-style: dotted;"/>
+                </we-button>
+                <we-button title="Double" data-customize-css-property="double">
+                    <div class="o_we_fake_img_item o_we_border_preview" style="border-style: double; border-left: none; border-right: none;"/>
+                </we-button>
+            </we-select>
+            <we-colorpicker data-customize-css-property=""
+                            data-css-property="border-top-color"
+                            data-selector-text=".s_hr > hr"
+                            data-no-transparency="true"
+                            data-color-prefix="border-"/>
+        </we-row>
+        <we-select string="⌙ Width" data-selector-text=".s_hr > hr" data-css-property="width">
+            <we-button data-customize-css-property="25%">25%</we-button>
+            <we-button data-customize-css-property="50%">50%</we-button>
+            <we-button data-customize-css-property="75%">75%</we-button>
+            <we-button data-customize-css-property="100%">100%</we-button>
+        </we-select>
```

Comment:
I skipped over these lines as I suspect they will probably be changed during a UX redesign anyway ... :p

Reply:
:eyes: